flutter_local_notifications/example/lib/main.dart

```diff
@@ -1960,6 +1960,7 @@ class _HomePageState extends State<HomePage> {
                 children: <Widget>[
                   Text(
                     'id: ${activeNotification.id}\n'
+                    'group: ${activeNotification.groupKey ?? 'no group'}\n'
```

Comment:
nit: for consistency, could the null coalescing be removed? The example will already display "null" when a notification doesn't have a title or body

Reply:
ok, i see.